my.css

```diff
@@ -0,0 +1,130 @@
+.header {
+	position: relative;
+	height: 100px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)